### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by arosser**
[IK-119] Styling for the route buttons on the homepage has been finalized. 3 temp fake routes were left for display. Will need to link real routes from DB once route display activity is done.

### CODE REVIEW

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 1]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
-     * Displays the unmatched routes of the user
+     * Creates and returns a button displaying route name, type and status
+     * @param name
+     * @param isDriver
+     * @param status
+     * @return Button
      */
-    public void createUnmatchedRoutes() {
-        LinearLayout linearLayout = (LinearLayout) findViewById(R.id.layoutScrollUnmatch);
+    public Button createRouteButton(String name, Boolean isDriver, int status){
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: space missing after closing paranthesis

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 2]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+        return new BitmapDrawable(getResources(), bitmapResized);
+    }
 
+    /**
+     * Sets appropriate image on the left side of the button
+     * @param button
+     * @param isDriver
+     * @return Button
+     */
+    private Button setImage(Button button, boolean isDriver){
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: same issue : space

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 3]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+        if(isDriver){
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: same issue : space

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 4]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            Drawable driverImg = getResources().getDrawable(R.drawable.wheel);
+            driverImg = resize(driverImg);
+            button.setCompoundDrawablesWithIntrinsicBounds(driverImg, null, null, null);
+        }
+        else{
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: same issue : space

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 5]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+     * @return drawable
      */
-    public LinearLayout.LayoutParams styleButton(Button button) {
-        LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(
-                LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT);
-        params.setMargins(0, 30, 0, 0);
-        button.setLayoutParams(params);
-        button.setBackgroundColor(getResources().getColor(R.color.backgroundColor));
-        return params;
+    private Drawable getRouteDrawable(int status){
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: same issue : space

File: client/app/src/main/java/com/example/mrides/Activity/HomePage.java [hunk 6]
```diff
@@ -110,45 +98,81 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+        Drawable drawable;
+            if(status == 0) {
```
Comment (adrielFab) [inline client/app/src/main/java/com/example/mrides/Activity/HomePage.java]: Add an inline comments for now that defines what status is

**Review by adrielFab — CHANGES_REQUESTED** [review]

**Review by SaifMahabub — APPROVED** [review]
Everything seems to be good.

**Review by adrielFab — APPROVED** [review]

**Pull request merged**